NEXT_CHANGES.rst

```diff
@@ -20,8 +20,53 @@ techniques
 - Added MSInlet``gas_flux_calibration_curve`` to enable multiple point calibration using calculated gas flux 
 either with different concentrations in carrier gas or at different inlet pressures. Note, concentration needs to be given in ppm, as the flux calculation uses various constants from the carrier gas molecule instead of a mixture, which will lead to significant inaccuracy for high concentrations.
 
+readers
+^^^^^^^
+``Mesurement.read_set`` can now take a folder as the first argument, in which case  all
+files in that folder (with the specified suffix) are appended.
+Resolves `Issue #88 <https://github.com/ixdat/ixdat/issues/88>`_
+``Measurement.read_set`` now also raises a ``ReadError` rather than returning ``None`` in
+the case of no matching files.
+
+``Measurement.read`` (and by extension ``Measurement.read_set``) can now be called
+without a specified reader for certain known file types. To see which file types, use::
```

Comment:
double ":" at the end of the line

Reply:
This is intentional. It is a ReStructuredText syntax.
https://docutils.sourceforge.io/docs/user/rst/cheatsheet.txt